taskcluster/ci/train-backwards/kind.yml

```diff
@@ -74,8 +75,11 @@ tasks:
             using: run-task
             command:
                 - bash
-                - -c
+                - -cx
                 - >-
+                    pip3 install --upgrade pip setuptools &&
+                    pip3 install -r $VCS_PATH/pipeline/train/requirements/train.txt &&
+                    export PATH="$HOME/.local/bin:$PATH" &&
```

Comment:
What is in `.local/bin`?

Reply:
This is where pip installs to without a virtualenv, I believe.

Reply:
opustrainer-train is there